index.js

```diff
@@ -200,6 +198,8 @@ class Session {
   }
 }
 
+module.exports.Session = Session
```

Comment:
I don't think `new Session()` is documented in the readme. Can you add it?

Reply:
Do you actually want to expose it as user facing API? I think for general use consumers of this library should try to let it own the session's lifecycle and thus should never need to instantiate one themselves. Even for library integrations (using `decodeSecureSession`) or what have you those consumers would get a `Session` instance from the `decodeSecureSession` function as opposed to creating one themselves.

I added it to the exports to be able to access it during testing as a simple cheat. Happy to document it if necessary but I think it might actually confuse things.

Reply:
If we expose `encodeSecureSession` then we should expose this as well. Honestly I would prefer not to.. maybe we can expose a `createSecureSession`.

Reply:
Ok, added